librarian_packages/hera_librarian/hera_librarian/__init__.py

```diff
@@ -76,14 +79,14 @@ def _do_http_post(self, operation, **kwargs):
 
         """
         kwargs['authenticator'] = self.config['authenticator']
-        for k in kwargs.keys():
+        for k in list(kwargs.keys()):
```

Comment:
This is `2to3` being paranoid, but you can skip the `list()` wrap here since you're not saving the output of `.keys()` anywhere.

Reply:
Good point, I removed it.